### ISSUE TITLE
Failed to install (1.4.0)

### ISSUE THREAD

**Issue by EricKotato** [adifyr/obsidian-chat-view#21]
For some reason I can't install latest version of plugin (1.4.0) to latest Obsidian (1.1.9). Both update and reinstall are failing with error:

```
app.js:1 Error: Request failed, status 404
    at new t (app.js:1:710073)
    at Aw (app.js:1:710265)
    at app.js:1:710942
    at app.js:1:235505
    at Object.next (app.js:1:235610)
    at a (app.js:1:234349)
(anonymous) @ app.js:1
```

**Comment by EricKotato** [comment]
I've managed to install 1.4.0 through BRAT, but there's something I've noticed.
![](https://user-images.githubusercontent.com/2903496/210288122-110fff22-c6db-435b-870b-94d7a49fb472.png)
Obsidian lists version as 1.5.0, though latest release is 1.4.0. Looks like version is fetched from master branch (probably from default branch?), but since there's no released tagged 1.5.0, it can't be downloaded.

**Comment by adifyr** [comment]
Hi @EricKotato. I've just released Version 1.5.0 of Obsidian Chat View. That should fix the error your getting. Let me know if it fixed it, and I'll close this issue.

**Comment by EricKotato** [comment]
Yes, plugin is properly updated now. Thanks.
